mod/groups/icon.php

```diff
@@ -1,57 +1,26 @@
 <?php
+
 /**
  * Icon display
  *
  * @package ElggGroups
  */
+elgg_deprecated_notice('icon.php has been depreated and should not be included. Use elgg_get_inline_url() instead.', '2.1');
 
-if (!isset($page)) {
-	die('This file cannot be called directly.');
-}
+$guid = get_input('group_guid');
+$size = get_input('size') ? : 'medium';
```

Comment:
true (i think), but if i for some stupid reason have a custom size ```0``` i could never get it ;)
what would be the best way to solve this?